packages/lsp4-contracts/package.json

```
@@ -46,7 +46,7 @@
     "lint:solidity": "solhint 'contracts/**/*.sol' && prettier --check 'contracts/**/*.sol'"
   },
   "dependencies": {
-    "@erc725/smart-contracts": "^7.0.0",
+    "@erc725/smart-contracts-v8": "file:erc725-smart-contracts-v8-rc0.tgz",
```

using local package here 

```suggestion
    "@erc725/smart-contracts": "^7.0.0",
```

---

Yes it is done on purpose, we need to use the ERC725 smart contracts from this PR. (Put as local package as it is not released yet).

https://github.com/ERC725Alliance/ERC725/pull/253